source/admin_manual/list_of_events.rst

```diff
@@ -1349,6 +1353,73 @@ Intermediate event emitted when an HTTP request is being retried.
 
 The ``http_request_finished`` event is still sent at the end of the request.
 
+
+HTTP Server
+===========
+
+These events are emitted by Dovecot's internal HTTP library when serving
+requests (e.g. doveadm HTTP API).
+
+Common fields
+-------------
+
+Fields present in all HTTP server events.
+
++---------------------+------------------------------------------------------+
+| Field               | Description                                          |
++=====================+======================================================+
+| Inherits from :ref:`event_incomming_conn`                                  |
++---------------------+------------------------------------------------------+
+| request_id          | Assigned ID if of the received request.              |
++---------------------+------------------------------------------------------+
+| method              | HTTP verb used uppercased, e.g. ``GET``.             |
++---------------------+------------------------------------------------------+
+| target              | Request path with parameters, e.g.                   |
+|                     | ``/path/?delimiter=%2F&prefix=test%2F``.             |
++---------------------+------------------------------------------------------+
+
+http_server_request_started
+---------------------------
+
+.. versionadded:: v2.3.17
+
+Emitted when a new HTTP request is received and the request headers
+(but not body payload) are parsed.
+
+http_server_request_finished
+----------------------------
+
+.. versionadded:: v2.3.17
+
+Emitted when the HTTP request's body is fully read and the response is about
+to be sent.
```

Comment:
Would be good for @stephanbosch to verify that the documentation matches the code in these event descriptions.